### PR TITLE
Update react-webcam

### PR DESCRIPTION
**Pull request by skubarenko**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:
If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: 
       - [React Webcam Props](https://github.com/mozmorris/react-webcam/blob/a354b668e4719533a491fde7fe6b907b56dcb956/src/react-webcam.js#L85-L102);
       - [React Webcam defaultProps](https://github.com/mozmorris/react-webcam/blob/a354b668e4719533a491fde7fe6b907b56dcb956/src/react-webcam.js#L74-L83);
- [x] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Review by squat — COMMENTED** [review]
Sorry for the delay, I am traveling at the moment. This looks overall like an improvement, but I am little confused by the new `defaulProps` field.

**Review by squat — CHANGES_REQUESTED** [review]
Sorry for the delay, I am traveling at the moment. This looks overall like an improvement, but I am little confused by the new `defaulProps` field.

File: types/react-webcam/index.d.ts [hunk 1]
```diff
@@ -11,24 +12,38 @@ export = Webcam;
 declare class Webcam extends React.Component<Webcam.WebcamProps, Webcam.WebcamState> {
     private static mountedInstances: Webcam[];
     private static userMediaRequested: boolean;
-    getScreenshot(): string|null;
-    getCanvas(): HTMLCanvasElement|null;
+    static defaultProps: Webcam.WebCamDefaultProps;
```
[the remark this answers is not in the document]
Comment (skubarenko) [reply types/react-webcam/index.d.ts]: These properties are defined in [here](https://github.com/mozmorris/react-webcam/blob/a354b668e4719533a491fde7fe6b907b56dcb956/src/react-webcam.js#L74-L83)
TypeScript 3.0 has added support for `defaultProps`, so I decided to add these types to the `d.ts`. Maybe in the future, IDEs will show these `defaultProps` in intellisense, but for now, you can navigate to type definitions and find the default values for component properties

[the remark this answers is not in the document]
Comment (squat) [reply types/react-webcam/index.d.ts]: Ah awesome. I had not realized this had been added.

**Pull request closed without merging**